<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>608</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>960</width>
    <height>550</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>200</width>
      <height>551</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>549</height>
      </rect>
     </property>
     <widget class="QPushButton" name="loadVideoButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Icons.qrc">
        <normaloff>:/resources/icons/resources/icons/48_addvideo.png</normaloff>:/resources/icons/resources/icons/48_addvideo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>44</width>
        <height>44</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="loadVideoText">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>30</y>
        <width>136</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>insert file path...</string>
      </property>
     </widget>
     <widget class="QLabel" name="browseVideoDescription">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>5</y>
        <width>136</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse for Video...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTabWidget" name="subjectTab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>200</width>
        <height>471</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Subjects</string>
       </attribute>
       <widget class="QPushButton" name="removeGroupButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>115</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addGroupButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>115</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QListWidget" name="groupsListWidget">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>25</y>
          <width>190</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="addGroupsLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>5</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Groups by Color Picker</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="subjectSelectorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>150</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/resources/icons/resources/icons/48_selector.png</normaloff>:/resources/icons/resources/icons/48_selector.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="groupClusterPickerButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>122</x>
          <y>150</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/resources/icons/resources/icons/48_color_picker.png</normaloff>:/resources/icons/resources/icons/48_color_picker.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="subjectsListWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>235</y>
          <width>190</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="addSubjectsLabel">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>210</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Subjects</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="removeSubjectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>330</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addSubjectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QDial" name="subjectAngleDial">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>350</y>
          <width>50</width>
          <height>64</height>
         </rect>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="subjectAngleLine">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>410</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="subjectColorLabel">
        <property name="geometry">
         <rect>
          <x>28</x>
          <y>375</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Regions</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/resources/icons/resources/icons/48_play.png</normaloff>
      <activeon>:/resources/icons/resources/icons/48_pause.png</activeon>
      <selectedon>:/resources/icons/resources/icons/48_play.png</selectedon>:/resources/icons/resources/icons/48_play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="frameBackward">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/resources/icons/resources/icons/48_frameback.png</normaloff>:/resources/icons/resources/icons/48_frameback.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="frameForward">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>500</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/resources/icons/resources/icons/48_frameforward.png</normaloff>:/resources/icons/resources/icons/48_frameforward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>510</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="frameIndexLabel">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>500</y>
      <width>200</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame: -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoFrame" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mousePosLabel">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>540</y>
      <width>75</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>X: - Y: -</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit_Ctrl_Q"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit_Ctrl_Q">
   <property name="text">
    <string>Quit [Ctrl Q]</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoFrame</class>
   <extends>QLabel</extends>
   <header location="global">videoframe.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
